provisioning/templates-api/local.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+set -Eeuo pipefail
+
+# Prevent direct run of the script
+if [ "${BASH_SOURCE[0]}" -ef "$0" ]
+then
+    echo 'This script should not be executed directly, please run "deploy_local.sh" instead.'
+    exit 1
+fi
+
+# Start minikube if needed
+if ! minikube status > /dev/null; then
+    echo
+    echo "Starting minikube ..."
+    echo "--------------------------"
+    minikube start
+fi
+
+# Build Docker image in the local Docker, so it is cached, if Minikube is destroyed
+IMAGE="$TEMPLATES_API_REPOSITORY:$TEMPLATES_API_IMAGE_TAG"
+echo
+echo "Building API image ..."
+echo "--------------------------"
+docker build -t "$IMAGE" -f "./docker/Dockerfile" "../../"
```

Comment:
Image buildujeme cez Docker, nie priamo v MiniKube.
Lebo ked sa zavola `minikube delete`, tak to zmaze aj build cache, 
a teda novy deploy trva zbytocne dlho.